suite-native/biometrics/src/useBiometrics.tsx

```diff
@@ -91,14 +91,38 @@ export const useBiometrics = () => {
             setAppStateVisible(appState.current);
         });
 
-        return () => subscription.remove();
+        return () => {
+            changeSubscription.remove();
+        };
     }, [
         isBiometricsOptionEnabled,
         setIsUserAuthenticated,
         setIsBiometricsOverlayVisible,
         isUserAuthenticated,
     ]);
 
+    /* Android does not recognizes the `inactive` app state. So to make the biometrics overlay visible when 
+       the app is out of focus on Android, we need to listen to the `blur` and `focus` Android specific events. */
```

Comment:
Could be something like: 
App state is not changed early enough for the screenshot and overlay rendering happens so we need to catch it by blur event

Reply:
Comment changed in 335eb3d80